python/apertium_core.i

```diff
@@ -120,17 +36,101 @@ apertium::postchunk_text(char arg, char *transferfile, char *datafile, char *inp
   free((char *) $1);
 }
 
-class apertium: public Transfer, public Interchunk, public Postchunk
+%inline%{
+#define SWIG_FILE_WITH_INIT
+#include <apertium/interchunk.h>
+#include <apertium/pretransfer.h>
+#include <apertium/postchunk.h>
+#include <apertium/tagger.h>
+#include <apertium/transfer.h>
+
+/**
+ * Imitates functionality of apertium-core binaries using file path
+ */
+
+
+void pretransfer(char arg, char *input_path, char *output_path)
 {
-public:
-  void interchunk_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
-  void pretransfer(char arg, char *input_path, char *output_path);
-  void postchunk_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
-  void transfer_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
+  bool useMaxEnt = false;
+  FILE *input = fopen(input_path, "r"), *output = fopen(output_path, "w");
+  processStream(input, output, false, false, false);
+  fclose(input);
+  fclose(output);
+}
+
+class transfer: public Transfer
```

Comment:
This isn't great :( Any alternatives?

Reply:
`apertium_transfer`?

Reply:
Usually classes are UpperCamelCase. Maybe `ApertiumTransfer`?

Reply:
Should I rename `tagger` -> `ApertiumTagger`